Makefile.PL

```diff
@@ -27,10 +27,10 @@ build_requires( 'Test::More' => 0, );
 # Dirty hack to avoid complaints about unparsable version number from
 # old versions of ExtUtils::MakeMaker.
 if ( eval { ExtUtils::MakeMaker->VERSION(6.68) } ) {
-    requires 'Zonemaster' => 'v1.0.4';
+    requires 'Zonemaster::Engine' => 'v1.0.4';
 }
 else {
-    requires 'Zonemaster' => 0;
+    requires 'Zonemaster::Engine' => 0;
```

Comment:
1. Is the "dirty hack" really needed? Isn't 1.000004 = v1.0.4 in the perl world?
2. I think we should use the real version of Zonemaster Engine, at least at the time of release. I guess it will be v2.0.0.